src-java/northbound-service/northbound-api/src/main/java/org/openkilda/northbound/dto/v2/yflows/YFlowUpdatePayload.java

```diff
@@ -49,6 +49,7 @@
     boolean strictBandwidth;
     String description;
     boolean allocateProtectedPath;
+    String diverseFlowId;
```

Comment:
Is it a flow or y-flow? Shouldn't be diverseYFlowId?

Reply:
As we discussed offline, this can be either flow or y-flow.